backend/game.js

```diff
@@ -603,25 +476,23 @@ module.exports = class Game extends Room {
     title: ${this.title}
     seats: ${this.seats}
     type: ${this.type}
-    sets: ${this.sets}
     isPrivate: ${this.isPrivate}
     picksPerPack: ${this.picksPerPack}
-    modernOnly: ${this.modernOnly}
-    totalChaos: ${this.totalChaos}
-    chaosPacksNumber: ${this.chaosPacksNumber}
-    packsInfos: ${this.packsInfo}
     players: ${this.players.length} (${this.players.filter(pl => !pl.isBot).map(pl => pl.name).join(", ")})
     bots: ${this.bots}
-    ${this.cube ?
-    `cubePoolSize: ${this.cube.cubePoolSize}
-    packsNumber: ${this.cube.packs}
-    playerPackSize: ${this.cube.cards}
-    cube: ${truncate(this.cube.list, 30)}`
-    : ""}`;
+    poolBuilder:
+    ${this.poolBuilder.toString()}`;
   }
 
   getNextPlayer(index) {
-    const {length} = this.players;
+    const { length } = this.players;
     return this.players[(index % length + length) % length];
   }
+
+  // Accessor required for unit test "can make a draft with 4 sets"
+  // (Note: It's weird that the unit tests evaluate private class members instead of
```

Comment:
The test that is validating this struck me as very weird for how I am used to thinking about and designing unit tests.

For a class/project like this I would have either expected to see unit tests that are testing the entire backend as a unit
(i.e. the unit tests would send fake messages that we expect to receive from the front-end, and validate that the response messages meet the expectations)

and/or

I would expect to see each class/module in the backend treated as a unit
(i.e. injecting mocks of the classes these components depend on, then testing the public API of the class/module while setting expectations on the mocks to stress the expected behaviors of each component in isolation)

(That said my unit testing experience is all on much lower level languages (C/C++) and more safety critical applications, so I'm not sure what the conventions are up here in web land.)

I'm not proposed any major overall of the unit tests. I don't see that as a very practical option; but I think the 2nd approach I listed there can work pretty well on this class with the poolBuilder abstraction I've proposed in the push request going forward? Curious for others' thoughts on this.